gcalbot/gcalbot/db.go

```diff
@@ -204,6 +215,29 @@ func (d *DB) GetChannelListByAccountID(accountID string) (channels []*Channel, e
 	return channels, nil
 }
 
+func (d *DB) GetExpiringChannelList() (channels []*Channel, err error) {
+	// query all channels that are expiring in less than a day
+	rows, err := d.DB.Query(`
+		SELECT channel_id, account_id, calendar_id, resource_id, ROUND(UNIX_TIMESTAMP(expiry)), next_sync_token FROM channel
```

Comment:
might be good to limit here

Reply:
Good point. Currently the renew process runs every hour, and renews all channels that expire in less than a day. If we put a limit here, how would you suggest we ensure that no channels expire without renewal? Some sort of paging?

Reply:
Actually might be better to just have a key on expiry so we don't end up with a table scan and we can page through if it becomes a bottleneck

Reply:
can you add this index?